gatsby/config-options/constants.ts

```diff
@@ -1,10 +1,11 @@
-import { buildGatsbyCloudPreviewUrl } from '../util/build-gatsby-cloud-preview-url';
+import { buildNetlifyPreviewUrl } from '../util/build-netlify-preview-url';
 export const RSS_FEED_URL = '/blog/rss.xml';
 export const DEFAULT_META_IMAGE_URL_PATH = '/sy-share-image.jpg';
-export const GATSBY_SITE_PREFIX = process.env.GATSBY_SITE_PREFIX || '';
-export const BRANCH_PREVIEW_URL = buildGatsbyCloudPreviewUrl({
-  prefix: GATSBY_SITE_PREFIX,
+export const NETLIFY_DOMAIN_NAME = process.env.GATSBY_NETLIFY_DOMAIN_NAME || '';
```

Comment:
where is this env coming from? netlify itself or something we have to provide? Did you change the netlify settings?

Reply:
I have configured it in the netlify settings.